src/main/java/com/uwetrottmann/tmdb2/entities/DiscoverFilter.java

```diff
@@ -22,7 +22,13 @@ public DiscoverFilter(Integer... items) {
         this.items = items;
     }
 
+    public DiscoverFilter(Separator separator, Integer... items) {
```

Comment:
Possibly change the existing constructor (`DiscoverFilter(Integer... items)`) to call this one?